### DISCUSSION TITLE
Missing link to organization home page from project pages

### DISCUSSION THREAD

**Discussion by douglaslassance** [community/community#6560] [Projects and Issues]
There is no apparent way to browse back to the organization home page from the project page (nor one to go back to the projects list page). I find this problematic navigation-wise.

**Comment by pietrodicaprio** [comment]
What is missing is the entire "header" of the organization

![telegram-cloud-photo-size-4-6039765651280738346-y](https://user-images.githubusercontent.com/37634431/139116592-fcc2fbdb-7335-4835-974a-bea1a1c4ef17.jpg)

**Comment by volkantash** [comment]
Bir örnek tasarım (A example desing)
Güncellendi (Updated)
![resim](https://user-images.githubusercontent.com/25375593/139251315-12c25233-fd67-4c25-b8fb-5b5ea3c6b524.png)

**Reply by mirkobrombin** [reply]
> Bir örnek tasarım (A example desing) Güncellendi (Updated) ![resim](https://user-images.githubusercontent.com/25375593/139251315-12c25233-fd67-4c25-b8fb-5b5ea3c6b524.png)

Personally I prefer the old header, much more intuitive, faster and easy to use.

**Reply by volkantash** [reply]
@mirkobrombin, CTRL + K ile açılan buyruklama bölümü işimizi görebilir. Gereksiz yükseklik oluşturmayalım. (The command section opened with CTRL + K can work. Let's not create unnecessary height.)

**Reply by mirkobrombin** [reply]
> @mirkobrombin, CTRL + K ile açılan buyruklama bölümü işimizi görebilir. Gereksiz yükseklik oluşturmayalım. (The command section opened with CTRL + K can work. Let's not create unnecessary height.)

Still less immediate than an easy header :D

**Reply by mirkobrombin** [reply]
Maybe something like this, taking advantage of free (unused?) space. (Not a concept, just copypaste from old manager to the new one).

![image](https://user-images.githubusercontent.com/9479903/139281981-b27bff26-a520-4494-b388-1d1e5015044b.png)

**Reply by volkantash** [reply]
Üzgünüm. Bu tasarım UI/UX dışı (Sorry. This desing is out of UI/UX). Çünkü, sekmeler 2021.11.14 adlı yerin sekmeleri olur. Üzerinde çalışmalısınız. (Because the tabs would be the tabs of 2021.11.14. You should work on it.)

**Reply by mirkobrombin** [reply]
> Üzgünüm. Bu tasarım UI/UX dışı (Sorry. This desing is out of UI/UX). Çünkü, sekmeler 2021.11.14 adlı yerin sekmeleri olur. Üzerinde çalışmalısınız. (Because the tabs would be the tabs of 2021.11.14. You should work on it.)

It was just an idea(not a concept as I wrote), there is a sure way to fit the old header into it without forcing users to do multiple clicks for something that was so simple before. I can study a concept but I'm not here for this purpose, mine is just a criticism :D

**Reply by mirkobrombin** [reply]
Btw I have no time for a full concept (that is actually my job :S) but I don't think these `30px` are a problem, especially if the lack of a practical menu slows down people's work. Also on small displays it can be hidden and/or the project cards can sticky on top at scroll. There are many methods and I'm sure the guys at GitHub don't want to give up a feature that is so (for me but I read for others too) important 😂🖖
![gh-header-refactoring](https://user-images.githubusercontent.com/9479903/139297736-f91757d9-742f-48e0-b311-f6656b456a87.png)

**Reply by volkantash** [reply]
Selâm. (Hi).

Yine aynı sorunlu tasarım. (Again the same problematic design.) Lütfen aşağıdaki kullanım düzeni gibi bir fikirle gelmeye çalışınız. (Please try to come up with an idea like the following usage pattern.)
Anlamak için. (for understand)
![resim](https://user-images.githubusercontent.com/25375593/139308328-1d522a7f-62ce-4231-98e8-24b6105825ee.png)

![resim](https://user-images.githubusercontent.com/25375593/139308986-51aee2ea-716e-4817-89eb-f0881be0390f.png)

**Comment by Jon1776** [comment]
I agree we need better navigation.  The new projects doesn't have a convenient way to get around yet.

**Comment by mkungla** [comment]
Collected all related discussions here: https://github.com/github/feedback/discussions/10448
